gpsoauth/_version.py

```diff
@@ -1 +0,0 @@
-__version__ = '0.4.3'
```

Comment:
looks like there's still some references to this file in the root `__init__.py`. It'd be nice to keep the `__version__` attr too -- https://github.com/python-poetry/poetry/issues/144 has some discussion of that.

Reply:
Good catch! I'd rather not add `__version__` as I want to follow convention to have constants uppercase.

Reply:
it's a [pep 396](https://www.python.org/dev/peps/pep-0396/#specification) thing that I'm not sure people really do anymore, but I might alias it later just in case people were depending on it.

Reply:
I think you're right, I checked few packages and several of them do that. I'll restore it in one the next PRs.